### ISSUE TITLE
Cloning MFEM using git protocol requires public key

### ISSUE THREAD

**Issue by benzwick** [mfem/PyMFEM#89]
When I tried to install the `mfem-4.3-dev` branch as suggested in https://github.com/mfem/PyMFEM/issues/88#issuecomment-873424279 I got the following error:
```
calling ... git clone git@github.com:mfem/mfem.git
Cloning into 'mfem'...
Warning: Permanently added the RSA host key for IP address '52.64.108.95' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

subprocess.CalledProcessError: Command '['git', 'clone', 'git@github.com:mfem/mfem.git']' returned non-zero exit status 128.
```

I got it working by simply replacing git with https in setup.py:
```
 repo_releases = {"mfem": "https://github.com/mfem/mfem/archive/v4.2.tar.gz",
                  "metis": "http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz",
                  "hypre": "https://github.com/hypre-space/hypre/archive/v2.20.0.tar.gz",}
-repos = {"mfem": "git@github.com:mfem/mfem.git", }
+repos = {"mfem": "https://github.com/mfem/mfem.git", }
 
 rootdir = os.path.abspath(os.path.dirname(__file__))
 extdir = os.path.join(rootdir, 'external')
```
Note that when trying to do this manually on GitHub there is this message:

> You don't have any public SSH keys in your GitHub account. You can add a new public key, or try cloning this repository via HTTPS.

Should the change be made in setup.py to use https to avoid this problem?

**Comment by sshiraiwa** [comment]
Probably yes ;D

**Comment by sshiraiwa** [comment]
Closing this....Addressed in mfem-4.3-dev.
